hazelcast/src/main/resources/hazelcast-full-example.yaml

```diff
@@ -0,0 +1,2515 @@
+# Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is a full example hazelcast.yaml that includes all the
+# configuration elements and attributes of a Hazelcast member.
+#
+# To use this, rename it to hazelcast.xml and place it in
+# the directory where you start your Hazelcast member.
+#
+# To learn how to configure Hazelcast, please see the Reference Manual
+# at https://hazelcast.org/documentation/
+
+hazelcast:
+
+  # You can use import to load different Hazelcast declarative configuration files you prepared.
+  # You can import as many YAML files as you want and hence compose your Hazelcast configuration
+  # out of those YAML files. If you want to use import, it should be placed at the top of your
+  # Hazelcast YAML file right after the hazelcast node.
```

Comment:
Hazelcast(cap 1st letter)

Reply:
Here it's the name of the YAML configuration root node. 
So what about this:
`... after the "hazelcast" root node` ?

Reply:
you are right, i mis-read it.